### PR TITLE
Add custom item layout for promotions

### PR DESCRIPTION
**Pull request by solocommand**
Screen Shot 2019-05-17 at 1 02 15 PM

### CODE REVIEW

File: packages/common/components/content/blocks/promotion-item.marko [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    <@footer-left|{ block }|>
+      <endeavor-link href=content.linkUrl target="_blank">
+        ${content.linkText}
+      </endeavor-link>
+    </@footer-left>
+  </if>
+  <@footer-right|{ block }|>
+    <if(content.type === 'event')>
+      <span class=`${block}__content-event-dates`>
+        <endeavor-content-starts block=block obj=content />
```
Comment (zarathustra323) [inline packages/common/components/content/blocks/promotion-item.marko]: Should this be here?

File: packages/common/components/content/blocks/promotion-item.marko [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        <endeavor-content-ends block=block obj=content />
+      </span>
+    </if>
+    <else>
+      <cms-content-published obj=content />
```
Comment (zarathustra323) [inline packages/common/components/content/blocks/promotion-item.marko]: Same for this?

File: packages/common/components/content/blocks/item.marko [hunk 3]
```diff
@@ -5,53 +5,61 @@ $ const primarySection = getAsObject(content, 'primarySection');
...
-    options=input.imageOptions
-    position=input.imagePosition
-    use-placeholder=input.imageUsePlaceholder
-    width=input.imageWidth
-  />
-  <@header-left|{ block }|>
-    <cms-website-section-name tag="span" block=block obj=primarySection link=true />
-    <if(input.withType)>
-      <cms-text-element tag="span" block=block obj=content field="typeTitled" />
+<if(content.type === 'contact')>
```
Comment (zarathustra323) [inline packages/common/components/content/blocks/item.marko]: Was the contact if statement here before?

Comment (solocommand) [reply packages/common/components/content/blocks/item.marko]: It wasn't, but IMO it should be?

Comment (zarathustra323) [reply packages/common/components/content/blocks/item.marko]: 👍

**Pull request merged**